xtask/src/runchecks.rs

```diff
@@ -130,7 +130,7 @@ fn cargo_fmt() {
 
 // Run cargo clippy command
 fn cargo_clippy() {
-    if std::env::var("RUST_MATRIX").map_or(false, |matrix| matrix != "stable") {
+    if std::env::var("CI_RUN").is_ok() {
         return;
     }
```

Comment:
So clippy won't run when `CI_RUN` is defined, because it will run with another github action. Is it correct?

Reply:
Exactly, only the `clippy-action` runs only on `stable` and `std` environment